explorer/interpreter/type_checker.cpp

```diff
@@ -353,9 +353,8 @@ auto TypeChecker::FieldTypesImplicitlyConvertible(
     llvm::ArrayRef<NamedValue> source_fields,
     llvm::ArrayRef<NamedValue> destination_fields,
     const ImplScope& impl_scope) const -> bool {
-  if (source_fields.size() != destination_fields.size()) {
-    return false;
-  }
+  // Todo: If default fields are implemented, the
+  //       code must be adapted to skip them
```

Comment:
```suggestion
  // TODO: If default fields are implemented, the
  // code must be adapted to skip them.
```

We do all-caps TODOs so that they're easier to grep for.